src/components/TokenAmount/TokenAmount.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useToken, UseTokenProvider } from 'use-token'
+import TokenAmount from 'token-amount'
+import { ImageExists } from '../../hooks'
+import { GU, textStyle } from '../../style'
+import { isAddress, warn } from '../../utils'
+import { useTheme } from '../../theme'
+
+const TokenAmountComponent = React.memo(function TokenAmountComponent({
```

Comment:
We should at least name our function `TokenAmount` so that the name appears correct in other places:

```suggestion
const TokenAmountComponent = React.memo(function TokenAmount({
```

But I was wondering if we should rename `TokenAmount` into `TokenAmountLib`, and just use `TokenAmount` for the component? :thinking:

Reply:
sounds good!